docker-compose.yml

```diff
@@ -11,11 +11,37 @@ services:
   fiks-protokoll-validator-api:
     image: fiks-protokoll-validator-api
     environment:
+      ASPNETCORE_URLS: "http://*:64558"
       ASPNETCORE_ENVIRONMENT: "LocalDev"
       ENVIRONMENT: "docker-compose-local-dev"
       fiks-protokoll-validator_ConnectionStrings__DefaultConnection: "Data Source=fiks-protokoll-validator-sqlexpress;Initial Catalog=fiks-protokoll-validator;User Id=SA;Password=Dev#FiksProtokollValidator1234"
       #fiks-protokoll-validator_ConnectionStrings__DefaultConnectionUserId: "SA"
       #fiks-protokoll-validator_ConnectionStrings__DefaultConnectionPassword: "Dev#FiksProtokollValidator1234"
     ports:
       - "127.0.0.1:64558:64558"
-      - "127.0.0.1:44303:44303"
\ No newline at end of file
+      - "127.0.0.1:44303:44303"
+  oidc-mock:
+    image: docker-all.artifactory.fiks.ks.no/oidc-provider-mock:1.0.2
+    ports:
+      - "127.0.0.1:8080:8080"
+  rabbitmq:
+    image: rabbitmq:3.7.13-management
+    volumes:
+      - /Users/jarle/KS/Repositories/gi-arkiv-2.0-validator/local/rabbitmq:/etc/rabbitmq
+    ports:
```

Comment:
Må du binde alle disse til localhost?

Reply:
Jeg hadde det for å kunne kjøre api i Docker og web "lokalt". Er nyttig når jeg skal utvikle på web?

Reply:
Var det et spørsmål? ;-)